src/static/js/almanac.js

```diff
@@ -700,9 +700,9 @@ function addShortKeyEventListers() {
     }
   });
 
-  // Prevent the search box from propogating key up events
+  // Prevent the search box and other inputs from propogating key up events
   // so the above are not activated when typing words with them.
-  document.querySelectorAll('.search-input').forEach(searchInput => {
+  document.querySelectorAll('input').forEach(searchInput => {
     searchInput.addEventListener("keyup", function onPress(event) {
       event.stopPropagation();
```

Comment:
Alternatively, WDYT about adding a check at the top of this event handler for something like this?

```js
if (isInputField(event.currentTarget)) {
  return;
}
```

Reply:
Will give it a go. Probably a bit more efficient - especially for our large DOM chapters!

Guessing I’ll need `target` though rather than `currentTarget`?

Reply:
Maybe, I get them mixed up and coded that off the top of my head :)

Reply:
Done!